sdk/lib/drivers/hidparser/parser.h

```diff
@@ -126,6 +127,7 @@ typedef struct
 
 }MAIN_ITEM_DATA, *PMAIN_ITEM_DATA;
 
+//#include <pshpack1.h>
```

Comment:
Is this because you're not sure? It's certainly not a hardware structure (it contains embedded pointers), so I think it's safe to completely remove that packing there.

If not then all the data before should be in a own (packed) structure, and that global item thing would then include that data as one member, then the embedded pointer.

Reply:
Oops, that is a test remainder :) Will remove